docs/content.zh/docs/dev/table/functions/udfs.md

```diff
@@ -1797,7 +1797,7 @@ public static class Top2 extends TableAggregateFunction<Tuple2<Integer, Integer>
 
 // 注册函数
 StreamTableEnvironment tEnv = ...
-tEnv.registerFunction("top2", new Top2());
+tEnv.createTemporarySystemFunction("top2", new Top2());
```

Comment:
ditto.

It is recommended to use class over new instance for udf that can be instanced with no parameter, because the platform can add more logic to control the process of creating new instance underneath, e.g. validation which is the built-in standard logic in TableEnvironmentImpl. More logic or optimization could be added in the future with no impact on users' existing code.